_drafts/2021-01-08-ALTF4-20.md

```diff
@@ -0,0 +1,78 @@
+
+*Description: What might sound obvious is actually not true: a single translator **can** translate a full issue by himself, and multiple translators don't always guarantee a translation at release*
+
+[Transition to next section, dunno am uncreative]
+
+### German Inefficiency
+
+I want to go into more detail with translating into German, as I think it's a great example for some of the benefits of having multiple people, but also highlights some challenges, that come up with organising the process or translating itself, and, let's be real, I just don't know that much about the other languages.
+
+We started as three, and after ironing out some initial technical hiccups, we were already going with the backlog of the first few issues. But then the first new issue came. For older issues it was fine for a single translator to work on it for however long it took him, but when the deadline is in 24 hours, we need to parallelise the work. Luckily it got apparent pretty soon that individual articles could be provided earlier than the whole issue, and with the added benefit of more available time we already got a split workload. But what do we do with our finished articles? Just sending them into our discord channel without any versioning proved pretty chaotic early on, and our own GitHub repo was on the table, but proved to big of a hassle for those without experience with git. So `_v1` version suffixes it was. with a (mostly) consistent naming scheme, and an [online difference checking tool](https://www.diffchecker.com/), we had everything we wanted from GitHub, but in our Discord channel.
+
+"Great, they know sorted everything, and that's it, no more problems, right?", you might think. Wrong! You might be right with technical issues, those mostly were sorted out indeed. But language isn't just 'right' or 'false', there are way more nuances[?] to it. You might have encountered different spellings in the english language between American, and British (and some others), those often only have one letter changed though, and even if they are to completely different words, a Brit will still understand what a [insert something american here] is. In German, however, there are some many words an Austrian uses everyday, but someone from northern Germany might never have heard them.
+
+Even though regional differences might seem like the biggest troublemaker, what kills us are actually the more subtle quirks, like the fact that we have a formal and a more informal version for 'you', and those sometimes completely change the structure of a sentence. In some settings, it's unambiguous on which form you should use, but especially for a blog, there are great arguments for both sides. (This is also the reason why I don't like dubbing, even big budget films get it wrong all the time, and it just feels soo unnatural when someone "Siezt" a close friend.)
```

Comment:
maybe adding an open to @Conor12345 most popular issue with FR genders? :D

Reply:
I'm not really sure what you mean with this, could you elaborate?